auth/src/main/java/com/hms/lib/commonmobileservices/auth/huawei/HuaweiAuthServiceImpl.kt

```diff
@@ -298,7 +299,7 @@ class HuaweiAuthServiceImpl : AuthService {
         return work
     }
 
-    override fun getPhoneCode(var1: String?, var2: String?): Work<Unit> {
+    override fun getPhoneCode(var1: String?, var2: String?, activity: Activity): Work<Unit> {
```

Comment:
please give the parameter names more clearly.

Reply:
resolved.

Reply:
not resolved. please give the parameter names more clearly instead of var1 value.